api/controllers/kosh.js

```diff
@@ -0,0 +1,71 @@
+const lib = require('../lib');
+
+exports.letter = async (req, res) => {
+  let conn;
+  const letter = req.params.Letter;
+  if (letter.length !== 1) {
+    lib.error(
+      'You must supply a single character to see the index for that letter.',
+      res,
+      413,
```

Comment:
I'm not sure if this is what 413 is really for, I think it's meant for size in terms of bytes. https://tools.ietf.org/html/rfc7231#section-6.5.11 `413 Payload Too Large` 

Would 422 `422 Unprocessable Entity` be be more appropriate? https://tools.ietf.org/html/rfc4918#section-11.2

Would like to hear other's opinions.

Reply:
i debated this a few times..